task/upload-sbom-to-trustification/0.1/upload-sbom-to-trustification.yaml

```diff
@@ -0,0 +1,231 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/tags: "sbom, trustification"
+  name: upload-sbom-to-trustification
+spec:
+  description: |-
+    Upload an SBOM file to [Trustification] using the [BOMbastic] API.
+
+    [Trustification]: https://github.com/trustification/trustification
+    [BOMbastic]: https://github.com/trustification/trustification/tree/main/bombastic
+
+    ## Configuration
+
+    This task requires some configuration and authentication secrets. By default, the task takes
+    them from a secret called `trustification-secret` that exists in the same namespace where the
+    task runs. You can override the secret name via the `TRUSTIFICATION_SECRET_NAME` param.
+
+    ### trustification-secret
+
+    Required keys:
+    - bombastic_api_url: URL of the BOMbastic api host (e.g. https://sbom.trustification.dev)
+    - oidc_issuer_url: URL of the OIDC token issuer (e.g. https://sso.trustification.dev/realms/chicken)
+    - oidc_client_id: OIDC client ID
+    - oidc_client_secret: OIDC client secret
```

Comment:
Where do RHTAP customers get the client ID and secret? Do they all use the single "walker" client created at install time?